.circleci/config.yml

```diff
@@ -1,61 +1,93 @@
 version: 2
 jobs:
   kinetic:
-    docker:
-      - image: autonomoustuff/docker-builds:kinetic-ros-base
+    machine:
+      image: circleci/classic:edge
+      docker_layer_caching: true
     steps:
       - checkout
       - run:
-          name: Set Up Container
+          name: Install Linuxcan on Host
           command: |
-            apt-add-repository -y ppa:jwhitleyastuff/linuxcan-dkms
-            apt-get update -qq && apt-get install -y linux-headers-generic linuxcan-dkms
-            source `find /opt/ros -name setup.bash | sort | head -1`
-            rosdep install --from-paths . --ignore-src -y
-            cd ..
-            catkin init
-            catkin config --extend /opt/ros/$ROS_DISTRO
+            sudo apt-add-repository -y ppa:jwhitleyastuff/linuxcan-dkms
+            sudo apt-get clean && sudo apt-get update -qq && sudo apt-get install dpkg # Fix dpkg bug
+            sudo apt-get install -y linux-headers-`uname -r` linuxcan-dkms
+            sudo depmod
+            sudo virtualcan.sh start
+            cd /usr/src/linuxcan-*/canlib/examples
+            ./listChannels
```

Comment:
is this just here to show that the install worked?

Reply:
Correct. There's no output from `sudo virtualcan.sh start` so I wanted something to show that the virtual channels actually existed in the logs.